guides/moving-to-oss.md

```diff
@@ -0,0 +1,227 @@
+[installation docs](https://coder.com/docs/coder-oss/latest/install) for more
+details. Something missing, or have feedback?
+[Let us know](https://coder.com/contact)</small>
+
+### CLI
+
+Coder OSS uses a seperate
+[command line utility](https://coder.com/docs/coder-oss/latest/install). To use
+both CLIs on the same machine, you can install the Coder OSS CLI under a
+different name (e.g. `codeross`):
+
+```sh
+curl -sL https://coder.com/install.sh | sh -s -- --method=standalone --binary-name=codeross > /dev/null
+
+# Coder v1 CLI
+coder workspaces list
+
+# Coder OSS CLI
+codeross list
+```
+
+### Users
+
+Like Coder v1, you can
+[enable SSO via OpenID Connect](https://coder.com/docs/coder-oss/latest/install/auth#step-2-configure-coder-with-the-openid-connect-credentials)
+so that any user in your federation can log in. Coder OSS optionally supports
+[GitHub (Enterprise)](https://coder.com/docs/coder-oss/latest/install/auth#step-1-configure-the-oauth-application-in-github)
+and [username/password](https://coder.com/docs/coder-oss/latest/users)
+authentication.
+
+> If you are interested in a bulk user and/or workspace migration utility,
+> [we'd like to hear from you](https://coder.com/contact).
+
+|                   | Coder v1 | Coder OSS                                                                                       |
+| ----------------- | -------- | ----------------------------------------------------------------------------------------------- |
+| Avatar            | ✅       | ❌                                                                                              |
+| Dotfiles          | ✅       | Per-workspace [(dotfiles docs)](https://coder.com/docs/coder-oss/latest/dotfiles)               |
+| Generated SSH key | ✅       | ✅                                                                                              |
+| Default shell     | ✅       | Per-workspace [(with parameters)](https://coder.com/docs/coder-oss/latest/templates#parameters) |
+| Auto-start times  | ✅       | Per-workspace                                                                                   |
+| Git OAuth         | ✅       | SSH key only                                                                                    |
+
+<small>Something missing, or have feedback?
+[Let us know](https://coder.com/contact)</small>
+
+User-wide settings (e.g. avatar, shell, autostart times, dotfiles URL) are not
+currently supported in Coder OSS
+[(#3506)](https://github.com/coder/coder/issues/3506).
+
+### Workspaces
+
+To migrate Coder v1 workspaces, you'll need at least one
+[template](https://coder.com/docs/coder-oss/latest/templates) in your Coder OSS
+deployment, specifically with the image(s) you support in Coder v1.
+
+> If you are interested in a bulk user and/or workspace migration utility,
+> [we'd like to hear from you](https://coder.com/contact).
+
+From there, we recommend manually creating a workspace in Coder OSS and using a
```

Comment:
here, do you mean that someone should first locally ssh into their coderv1 terminal and execute the following commands? make that more clear 

theres a bit of assumption going on around where people will be starting from at this point. imo we need to hand hold them to get to this point. what steps do they need to get to execute the following commands???

Reply:
> here, do you mean that someone should first locally ssh into their coderv1 terminal and execute the following commands? make that more clear

Yep - I can clarify